File: ruby/product.rb
def product (firstNumber, secondNumber, thirdNumber)
	return firstNumber * secondNumber * thirdNumber
end

puts("Enter the first number: ")
firstNumber = gets.to_i
puts("Enter the second number: ")
secondNumber = gets.to_i
puts("Enter the third number: ")
thirdNumber = gets.to_i
result = product(firstNumber, secondNumber, thirdNumber)
print "Product of #{firstNumber}, #{secondNumber} and #{thirdNumber} is #{result}."

File: perl/product.pl
sub product {
	($firstNumber, $secondNumber, $thirdNumber) = @_;
	$result = $firstNumber * $secondNumber * $thirdNumber;
	return $result;
}

print "Enter first number: ";
$firstNumber = <STDIN>;
#print "DEBUGG : First number is $firstNumber."
print "Enter second number: ";
$secondNumber = <STDIN>;
print "Enter third number: ";
$thirdNumber = <STDIN>;
$result = product($firstNumber, $secondNumber, $thirdNumber);
print "Product of $firstNumber, $secondNumber and $thirdNumber is $result.";